src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
+        {
+            pE.mArgumentList.append( QString::number(lua_toboolean( L, i )) );
```

Comment:
:bulb: I had the same idea a while back but never got around to finishing or publishing it (as part of an inter-profile event signalling system) - just goes to show - great minds think alike!

 As the arguments are handled as strings I choose to use "true" and "false" as the strings to pass and - in keeping with the 'C' standard of "anything that isn't zero is true" parsed any ARGUMENT_TYPE_BOOLEAN argument as:
`lua_pushboolean( L, pE.mArgumentList[i] != QStringLiteral( "false" ) )`
so that anything that WASN'T "false" becomes a bool *true*.  I felt this might make debugging in the Qt IDE a tiny bit easier - as it was easy to tell if an argument was a number or a boolean...

Any thoughts?